lib/inv_ca_gcr.cpp

```diff
@@ -4,16 +4,9 @@
 
 namespace quda {
 
-  enum Basis {
-    POWER_BASIS,
-    INVALID_BASIS
-  };
-
-  const static Basis basis = POWER_BASIS;
-
   CAGCR::CAGCR(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile)
     : Solver(param, profile), mat(mat), matSloppy(matSloppy), init(false),
-      alpha(nullptr), rp(nullptr), tmpp(nullptr), tmp_sloppy(nullptr) { }
+      basis(param.ca_basis), alpha(nullptr), rp(nullptr), tmpp(nullptr), tmp_sloppy(nullptr) { }
```

Comment:
clang-format suggested the following change:


```
-  CAGCR::CAGCR(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile)
-    : Solver(param, profile), mat(mat), matSloppy(matSloppy), init(false),
-      basis(param.ca_basis), alpha(nullptr), rp(nullptr), tmpp(nullptr), tmp_sloppy(nullptr) { }
+  CAGCR::CAGCR(DiracMatrix& mat, DiracMatrix& matSloppy, SolverParam& param, TimeProfile& profile)
+      : Solver(param, profile)
+      , mat(mat)
+      , matSloppy(matSloppy)
+      , init(false)
+      , basis(param.ca_basis)
+      , alpha(nullptr)
+      , rp(nullptr)
+      , tmpp(nullptr)
+      , tmp_sloppy(nullptr)
+  {
+  }
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_ca_gcr.cpp -->